src/views/ConsentContinuationOut.vue

```diff
@@ -945,5 +945,19 @@ h2 {
   .value.certifiedby {
     padding-left: 3rem;
   }
+
+  .certify-statement-section {
+    padding-left: 15rem;
+  }
+
+  @media (max-width: 768px) {
```

Comment:
For large screens:
![large](https://user-images.githubusercontent.com/122301442/234706303-644bf3d1-86e9-4e9a-b1d7-ec54a5202f45.PNG)

For small screens:
![small](https://user-images.githubusercontent.com/122301442/234706430-4a18c977-043f-42c1-9a5f-ab70a4b785e0.PNG)

Reply:
While this looks OK, it's reinventing the grid system and its built-in breakpoints/responsiveness.

Ref: https://v2.vuetifyjs.com/en/components/grids/

Reply:
Sev, I had a discussion with Ethan regarding the Certify component. Apparently, this design of the certify statement being on the same line as the text-field is needed everywhere in other local filings.

So I went ahead and refactored the Certify component's code and cleaned it up. It now uses the grid system.

Reply:
Looks awesome!